doc/spectrographs/gemini_gmos.rst

```diff
@@ -118,3 +123,35 @@ your RMS is a factor of 2-3 too high, consider setting:
             fwhm_fromlines = False
 
 
+MultiSlit
+=========
+
+Mask Definition
+---------------
+
+PypeIt can now take advantage of the mask definition file
+generated when one designs a GMOS mask.  To do so, one needs
+to provide two additional files and specify them 
+with the :doc:`pypeit_file`:
+
+#.  The mask definition file, aka ODF file
+#.  An aligment image (taken with the spectra)
```

Comment:
Perhaps add: "Ensure that the alignment stars in the image are centered in the mask's square alignment slits. i.e. choose the final image in the sequence of alignment images from the observations."

Reply:
added